contracts/serverless-v2/Relay.sol

```diff
@@ -217,25 +229,25 @@ contract RequestChainContract is
 
     event JobCancelled(uint256 indexed jobId);
 
-    error InvalidUserTimeout();
-    error JobNotExists();
-    error OverallTimeoutOver();
-    error InvalidJobOwner();
-    error OverallTimeoutNotOver();
+    error RelayInvalidUserTimeout();
+    error RelayJobNotExists();
+    error RelayOverallTimeoutOver();
+    error RelayInvalidJobOwner();
+    error RelayOverallTimeoutNotOver();
 
     //-------------------------------- internal functions start -------------------------------//
 
     // TODO: create deposits of USDC and native token
     function _relayJob(
         bytes32 _codehash,
         bytes memory _codeInputs,
-        uint256 _userTimeout,
+        uint256 _userTimeout,   // in milliseconds
         uint256 _maxGasPrice,
         uint256 _usdcDeposit,
         uint256 _callbackDeposit
     ) internal {
-        if(_userTimeout <= GLOBAL_MIN_TIMEOUT || _userTimeout >= GLOBAL_MAX_TIMEOUT)
-            revert InvalidUserTimeout();
+        if(_userTimeout <= (GLOBAL_MIN_TIMEOUT * 1000) || _userTimeout >= (GLOBAL_MAX_TIMEOUT * 1000))
```

Comment:
Let's keep the values of GLOBAL_MIN_TIMEOUT and GLOBAL_MAX_TIMEOUT in milliseconds, it will give more precision to set boundaries.

Reply:
keep everything except the attestation timestamp in seconds only